docs/platform-qemu.md

```diff
@@ -21,13 +43,23 @@ supports a number of different disk formats.
 
 ## Networking
 
-TBD
+The `qemu` backend supports a number of networking options, depending
+on the platform you are running. The default is the userspace
+networking which provides the VM with a internal DHCP server and
+network connectivity, but does not provide access to the VMs network
+from the outside.
+
+With user mode networking you can publish selected VM ports on the
+host, using the `-publish` option. It uses the same syntax as the
+`qemu` binary.
+
+On Linux, you can attach the VM either to an existing bridge or tap interface.
```

Comment:
You didn't fancy explaining how ;-) You could perhaps steal a bunch of useful words from the `linuxkit/kubernetes/README.md`?

Reply:
Indeed. But will take a look at the kube docs. Thanks for the reminder